dao/src/main/java/com/redhat/ipaas/dao/manager/DataManager.java

```diff
@@ -111,7 +111,12 @@ public void store(ModelData modelData) {
             if (!id.isPresent()) {
                 LOGGER.warn("Cannot load entity from file since it's missing an id: " + modelData.toJson());
             } else {
-                if (fetch(kind, id.get()) == null) {
+                WithId prev = null;
+                try {
+                    prev = fetch(kind, id.get());
+                } catch (RuntimeException e) {
```

Comment:
Why catching and ignoring ? A comment would be nice, explaining why this would make sense here.